docs/v0.9/en/user-guide/operations/compaction.md

```diff
@@ -0,0 +1,142 @@
+# Compaction
+
+For databases based on the LSM Tree, compaction is extremely critical. It merges overlapping fragmented SST files into a single ordered file, discards deleted data while significantly improves query performance.
+
+Until v0.9.1, GreptimeDB provides compaction strategies to control how SST files are compacted: Time Windowed Compaction Strategy (TWCS) and strict window compaction strategy.
```

Comment:
```suggestion
Until v0.9.1, GreptimeDB provides compaction strategies to control how SST files are compacted: Time Windowed Compaction Strategy (TWCS) and strict window compaction strategy (SWCS).
```

Reply:
fixed in https://github.com/GreptimeTeam/docs/commit/5328a81f0c42cdc40aecbf168efa50e455f3db0d#diff-2643784f62d50a7674027cfc0acf77e84d673b2407cff812d3f0a1d92962571cL5-R5